### DISCUSSION TITLE
How to Specify the Root of the Entire Directory as Source in Sync Config

### DISCUSSION THREAD

**Discussion by haroon-niazi** [BetaHuhn/repo-file-sync-action#279] [Q&A]
Hi,

I am trying to specify the sync config as:

```
- source: ./
  dest: dir-a/
  exclude: |
    .github/
```

The idea is to sync the root of the source directory with some exclusions.  But while doing above, I am getting **Cannot copy './' to a subdirectory of itself,**. I want to avoid updating the sync config every time there is a new file or folder added to the source repo. 

Any help would be highly appreciated.

**Comment by tbutler-qontigo** [comment]
It can't be done at the moment - see https://github.com/BetaHuhn/repo-file-sync-action/discussions/130
